docs/source/specs/openapi.json

```diff
@@ -54,6 +54,9 @@
                 ],
                 "summary": "Obtain server status",
                 "operationId": "getStatus",
+                "security": [{
```

Comment:
Status endpoint doesn't require authentication.

Reply:
Followed suggestion and deleted this, the endpoint does not ask for authentication any more